src/generators.rs

```diff
@@ -579,62 +569,64 @@ pub fn directed_grid_graph(
     })
 }
 
-// MAX_ORDER is determined based on the pointer width of the target platform
-#[cfg(target_pointer_width = "64")]
-const MAX_ORDER: u32 = 60;
-#[cfg(not(target_pointer_width = "64"))]
-const MAX_ORDER: u32 = 29;
-
-/// Generate an undirected binomial tree of order n recursively.
+/// Generate an undirected heavy square graph. Fig. 6 of
+/// https://arxiv.org/abs/1907.09528.
+/// An ASCII diagram of the graph is given by:
 ///
-/// :param int order: Order of the binomial tree. The maximum allowed value
-///     for order on the platform your running on. If it's a 64bit platform
-///     the max value is 59 and on 32bit systems the max value is 29. Any order
-///     value above these will raise a ``OverflowError``.
-/// :param list weights: A list of node weights. If the number of weights is
-///     less than 2**order extra nodes with with None will be appended.
-/// :param bool multigraph: When set to False the output
+/// .. code-block:: console
+///
+///     ...       S   ...
+///        \     / \
+///        ... D   D   D ...
+///            |   |   |
+///        ... F-S-F-S-F-...
+///            |   |   |
+///        ... D   D   D ...
+///            |   |   |
+///        ... F-S-F-S-F-...
+///            |   |   |
+///            .........
+///            |   |   |
+///        ... D   D   D ...
+///             \ /     \
+///        ...   S       ...
+///
+/// NOTE: This function generates the four-frequency variant of the heavy square code.
```

Comment:
Should we use `.. note::` here? Since this will be rendered in a sphinx context we can use the directive.